Makefile.js

```diff
@@ -197,7 +197,8 @@ function generateRuleIndexPage() {
                         name: basename,
                         description: rule.meta.docs.description,
                         recommended: rule.meta.docs.recommended || false,
-                        fixable: !!rule.meta.fixable
+                        fixable: !!rule.meta.fixable,
+                        hasSuggestions: !!rule.meta.docs.suggestion || false
```

Comment:
```suggestion
                        hasSuggestions: !!rule.meta.docs.suggestion
```

`|| false` seems unnecessary here since the left side is already a boolean?

Reply:
I think you're right since `!!undefined` is `false`. I was just following what `recommended` does. Fine to remove it.